lib/companies/schema/company.ex

```diff
@@ -1,27 +1,33 @@
 defmodule Companies.Schema.Company do
   @moduledoc false
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
   alias Companies.Industries
 
-  # @enforce_keys [:industry, :last_changed_on, :location, :name, :website]
+  schema "companies" do
```

Comment:
Odd thought, but what about adding a `file_path` attribute to the struct? This will allow us to add validation messages on it, as well as allowing us to do fun future things like adding a "Quick Edit" button in the web UI that points to GitHub edit page.